### ISSUE TITLE
How to change batch_size for training using plants_training code ?

### ISSUE THREAD

**Issue by brisker** [bernard24/RIS#9]
I used my own dataset and it has more than 11000 training images. I directly use  the plant_training code, training from scratch, but the training did not seem to converge. And at the beginning the training seemed weird too:

![image](https://cloud.githubusercontent.com/assets/13804492/20430898/0330c40e-add2-11e6-9b01-0c888b1d614c.png)

So is it caused by the batch_size? Should I change batch_size from 1 to 32 or more? Could you please give some advice?

Besides, 
1.--- In your paper you said you use one image for each batch in leaf segmentation experiments , so how to change the batch-size?  Do I need to modify MatchCriterion.lua to adapt to mini-batch mode?

2.---what is the batch_size for your coco dataset experiments?

**Comment by brisker** [comment]
@bernard24

**Comment by bernard24** [comment]
The most likely reason for what you are showing is that the learning rate is too big. I would try decreasing the learning rate by one or two orders of magnitude.
Regarding the batch size, I always assume it is 1 throughout the code. Changing that would require quite a few changes in many different files, and in any case, a batch size of more than 1 image may not fit in GPU memory.
